anubis-management-api/src/policies/routers.py

```diff
@@ -9,9 +9,6 @@
 from rego import serialize as r_serialize
 import default
 
-
-models.Base.metadata.create_all(bind=engine)
```

Comment:
ok to move this in main and split per model (even though I am not sure what's the rational for doing so)